tests/addresslib/address_test.py

```diff
@@ -138,7 +138,7 @@ def test_edge_cases():
 def test_display_name__to_full_spec():
     eq_('"foo (\\"bar\\") blah" <foo@bar.com>',
         EmailAddress('foo ("bar") blah', 'foo@bar.com').full_spec())
-    eq_('"foo. bar" <foo@bar.com>',
+    eq_('foo. bar <foo@bar.com>',
```

Comment:
I agree that it might make sense to be able to parse addresses like `foo. bar <foo@bar.com>`, but I do not understand why `full_spec` should produce obsolete syntax?